ci/scripts/prepare-cached-bitstream.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# This script currently assumes it is used for chip_earlgrey_cw310.
+
+set -e
+
+. util/build_consts.sh
+
+TOPLEVEL=top_earlgrey
+TARGET=cw310
+
+TOPLEVEL_BIN_DIR="$BIN_DIR/hw/$TOPLEVEL"
+mkdir -p ${TOPLEVEL_BIN_DIR}
+
+BOOTROM_SUFFIX=.scr.39.vmem
```

Comment:
It's actually not in CI. :stuck_out_tongue: 

That PR failed to make necessary updates all over. The FPGA splicing wouldn't work, nor would various other things in CI. A simple `git grep scr.39.vmem` reveals a number of remaining locations.

Reply:
If you're wondering _how_ CI could still work, it's because it doesn't care about bazel's file name. It uses target-location.sh to grab it, then copies it over to BIN_DIR with the old suffix :joy:

Reply:
oh lovely, yes I was wondering hehe. too bad we can't define this in one place and use it throughout (maybe a cleanup for another PR). marking resolved.